docs/examples/gnmi-clients/gnmic/index.md

```diff
@@ -82,13 +933,104 @@ gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure subscribe --path \
   "/interfaces/interface/state/counters"  >> outputs/interface_state.json
 ```
 
-## eos_native paths
+### EOS Native paths
+
+To get EOS native paths, OCTA has to be enabled as mentioned in the
+configuration section. Performing GET/SUBSCRIBE actions using EOS native paths
+require changing the origin to `eos_native`.
+
+#### Commonly used paths
+
+- MAC table: `/Smash/bridging/status/smashFdbStatus`
+- ARP table: `/Smash/arp/status/arpEntry`
+- Neighbor table: `/Smash/arp/status/neighborEntry`
+- IPv4 RIB: `/Smash/routing/status/route`
+- IPv6 RIB: `/Smash/routing6/status/route`
+- IPv4 next-hop table: `/Smash/routing/status/nexthop`
+- IPv6 next-hop table: `/Smash/routing6/status/nexthop`
+- CPU info: `/Kernel/proc/cpu`
+- Process statistics: `/Kernel/proc/stat`
+- System info: `/Kernel/sysinfo`
+- EOS version: `/Eos/image`
+- Interface counters: `/Smash/counters/ethIntf/<agent>/current/counter`
+  - Values for `<agent>` are:
+    - 7500-family, 7280-family, 7020-family (Arad/Jericho ASICs): `SandCounters`
+    - 7300-family, 7250-family, 7050-family, 7010 products, 720-family (Trident
+      ASICs): `StrataCounters`
+    - For 7060-family, 7260-family (Tomahawk): `Strata-FixedSystem` or
+      `StrataCounters` from 4.22+
+    - 7150-family products (Alta ASICs): `FocalPointV2`
+    - 7160-family products (Cavium/Xpliant ASICs): `XpCounters`
+    - 7170-family products (Barefoot ASIC): `BfnCounters`
+
+##### Get CPU utilization
 
 ```shell
-gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure get --path \
+ gnmic -a 10.83.13.108:6030 -u cvpadmin -p arastra --insecure --gzip get --path  \
+  'eos_native:/Kernel/proc/cpu/utilization/total'
+```
+
+<details><summary> Reveal output</summary>
+<p>
+
+```javascript
+[
+  {
+    "timestamp": 1626291721294738334,
+    "time": "2021-07-14T20:42:01.294738334+01:00",
+    "prefix": "eos_native:Kernel/proc/cpu/utilization/total",
+    "updates": [
+      {
+        "Path": "name",
+        "values": {
+          "name": "total"
+        }
+      },
+      {
+        "Path": "nice",
+        "values": {
+          "nice": 28845
+        }
+      },
+      {
+        "Path": "util",
+        "values": {
+          "util": 41
+        }
+      },
+      {
+        "Path": "user",
+        "values": {
+          "user": 32058347
+        }
+      },
+      {
+        "Path": "system",
+        "values": {
+          "system": 4597389
+        }
+      },
+      {
+        "Path": "idle",
+        "values": {
+          "idle": 76373566
+        }
+      }
+    ]
+  }
+]
+```
+
+</p>
+</details>
+
+##### Get transceiver DOM temperature
+
+```shell
+gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure --gzip get --path \
   eos_native:/Sysdb/hardware/archer/xcvr/status >> outputs/doms.json
 
-gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure get \
+gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure --gzip get \
   --path eos_native:/Eos/image >> outputs/eos_image.json
```

Comment:
might split this into a separate code block with descriptive text to match.  this isn't DOM output. 😉